Makefile

```diff
@@ -69,6 +69,9 @@ lint: golangci-lint ## Runs golangci-lint linter
 test: ## Runs go tests
 	go test ./...
 
+integration: ## Runs integration tests
+	bats --verbose-run tests/integration.sh
```

Comment:
for local run, we might want to check if bats is installed and fail if not (or install it in a custom directory)

Reply:
added a check for `bats`, is that OK?